importer.py

```diff
@@ -68,12 +68,13 @@ def tiff_to_jpg(self, tiff_filepath):
             raise ConvertException(f"Bad filename, can't convert {tiff_filepath}")
 
         jpg_dest = os.path.join(TMP_JPG, file_name_no_extention + ".jpg")
-
+        # 'exiftool', '-TagsFromFile', tiff_filepath, jpg_dest, '&&',
```

Comment:
this is just commented out working code, for trying to get exif to transfer on conversion.